packages/roosterjs-content-model/lib/modelApi/image/applyImageBorderFormat.ts

```diff
@@ -0,0 +1,50 @@
+import { Border, extractBorderValues } from '../../domUtils/borderValues';
+import { ContentModelImage } from '../../publicTypes/segment/ContentModelImage';
+import { ptToPx } from '../../formatHandlers/utils/pointsToPixels';
+
+/**
+ * @internal
+ */
+export default function applyImageBorderFormat(
+    image: ContentModelImage,
+    border: Border,
+    isPt?: boolean
+) {
+    const format = image.format;
+    const { width, style, color } = border;
+    const borderKey = 'borderTop';
+    const extractedBorder = extractBorderValues(format[borderKey]);
+    const borderColor = extractedBorder.color;
+    const borderWidth = extractedBorder.width;
+    const borderStyle = extractedBorder.style;
+    let borderFormat = '';
+
+    if (width) {
+        borderFormat = isPt ? ptToPx(parseInt(width)) + 'px' : width;
+    } else if (borderWidth) {
+        borderFormat = borderWidth;
+    } else {
+        borderFormat = '1px';
+    }
+
+    if (style) {
+        borderFormat = `${borderFormat} ${style}`;
+    } else if (borderStyle) {
+        borderFormat = `${borderFormat} ${borderStyle}`;
+    } else {
+        borderFormat = `${borderFormat} solid`;
+    }
+
+    if (color) {
+        borderFormat = `${borderFormat} ${color}`;
+    } else if (borderColor) {
+        borderFormat = `${borderFormat} ${borderColor}`;
+    }
+
+    image.format.borderRadius = '5px';
```

Comment:
hard code 5px? Is there any way to remove it by user? e.g. if I don't want any border, I should be able to remove this border radius as well